### DISCUSSION TITLE
Is there a way to replace a member options of a map in a scope?

### DISCUSSION THREAD

**Discussion by Kation** [LuckyPennySoftware/AutoMapper#4511] [NA]
I use `ProjectTo` to map ef core query to dto. And there is a nagivation collection map need to list dto.
I have to filter this collection using query string while request dynamic.
How to replace member options in this request scope only?

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/Queryable-Extensions.html#parameterization
